packages/riipen-ui/src/components/InputHint.jsx

```diff
@@ -45,6 +51,10 @@ class InputHint extends React.Component {
             color: ${theme.palette.text.secondary};
             margin-bottom: ${theme.spacing(2)}px;
           }
+
+          .white {
```

Comment:
Should there be a whitelist of available colours or nah?

Reply:
ah yea probably in the proptypes